os/include/setjmp.h

```diff
@@ -0,0 +1,86 @@
+/****************************************************************************
```

Comment:
I am sorry I saw it lately. This is working on arm only. Let's put it in os/arch/arm/include.

Reply:
After modify this, I will merge this.

Reply:
Why move anything?

There should be setjmp/longjmp for each arch/platform targeted by TizenRT (feature parity).

If TizenRT is arm only, does it matter where we put these headers?
If there are other architectures that TizenRT is potentially addressing, setjmp/longjmp should be provided for them as well. Which leads us to having setjmp.h in os/include directory.
Or at least there should be a compilation error when someone tries to include relevant header. In that case 
>\#include <arch/setjmp.h>

would spit error due to setjmp.h header not being provided for that particular arch.